app/routes/surveys/components/Submissions/Results.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+
+import React from 'react';
+import styles from '../surveys.css';
+import type { SurveyEntity, QuestionEntity } from 'app/reducers/surveys';
+import { QuestionTypes, CHART_COLORS } from '../../utils';
+import InfoBubble from 'app/components/InfoBubble';
+import { VictoryPie, VictoryTheme } from 'victory';
+
+type Props = {
+  survey: SurveyEntity,
+  graphData: Object,
+  numberOfSubmissions: number,
+  generateTextAnswers: QuestionEntity => any
```

Comment:
Guess this can be written using react-types

Reply:
What would that look like?